src/components/action-toolbar/action-toolbar.component.js

```diff
@@ -34,36 +34,19 @@ import './action-toolbar.scss';
  *
  *  Additional props for Link or Icon can be passed in the action object.
  *
- * @class ActionToolbar
- * @constructor
  */
 class ActionToolbar extends React.Component {
   // TODO This component needs to be freestanding - we need to provide an api that allows it be used independently.
   // https://github.com/Sage/carbon/issues/1070
 
   static propTypes = {
```

Comment:
Would you like to move the propTypes out of the static?
`ActionToolbar.propTypes = {}`